src/toil/fileStore.py

```diff
@@ -1603,7 +1607,8 @@ def open(self, job):
                                            disk=diskUsed,
                                            humanRequestedDisk=bytes2human(jobReqs),
                                            requestedDisk=jobReqs))
-            self.logToMaster(logString, level=logging.INFO)
+            self.logToMaster(logString, level=(logging.DEBUG if self.jobName.startswith(self.debugJobNames)
```

Comment:
Ok, I think I understand the intent now. In that case, there are a couple ways to easily improve tracking of the workflow. I'm still in favor of decreasing this statement to DEBUG since its main purpose is propagating the disk usage message which I think is only of general interest when the usage is greater than expected, which is handled by the next log statement.

Instead, I think increasing this message to INFO (potentially with the internal CWL message masking) will do what you are looking for: https://github.com/BD2KGenomics/toil/blob/36d22d1171075de4918b40a317dbdce921ea5f31/src/toil/leader.py#L423 

Bumping this message to INFO and improving it by changing ```jobNode.jobStoreID``` to just ``jobNode`` could also help track the workflow's progress: https://github.com/BD2KGenomics/toil/blob/36d22d1171075de4918b40a317dbdce921ea5f31/src/toil/leader.py#L519

With those changes you'll get messages when the task is ready to be run & when it is finished running.